src/components/DraggableContent/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { GrDrag } from 'react-icons/gr';
```

Comment:
Are we going to use different icon styles throughout the deep-ui?

Reply:
By the order of preference
1. Io5
2. Io4
3. Other